origin-contracts/package.json

```diff
@@ -47,9 +48,9 @@
     "chai-bignumber": "^2.0.2",
     "chalk": "^2.4.1",
     "cli-table": "^0.3.1",
-    "ganache-core": "^2.2.1",
-    "prettier-eslint-cli": "^4.7.1",
+    "ganache-core": "^2.2.0",
```

Comment:
Should we actually pin to 2.2.1 rather than use ^2.2.0 since that could cause 2.3.0 to get installed ?
Same comment for origin-js